src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,10 +6,15 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    /* Anakin Prefix definitions */
+    public static final Prefix PREFIX_NAME = new Prefix("n/");
+    public static final Prefix PREFIX_DECK = new Prefix("d/");
```

Comment:
Do any commands use /d ?

Reply:
`newcard` and `editcard` uses the prefix `d/` to indicate which deck to look at